client/state/sites/products/conflicts.ts

```diff
@@ -120,25 +135,29 @@ export const isBackupProductIncludedInSitePlan = createSelector(
 			return null;
 		}
 
-		let feature;
+		const DAILY_BACKUP_PRODUCTS = [
+			PRODUCT_JETPACK_BACKUP_DAILY,
+			PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY,
+		];
+
+		const REALTIME_BACKUP_PRODUCTS = [
+			PRODUCT_JETPACK_BACKUP_REALTIME,
+			PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+			PRODUCT_JETPACK_BACKUP_T2_YEARLY,
+			PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY,
+			PRODUCT_JETPACK_BACKUP_T1_MONTHLY,
+			PRODUCT_JETPACK_BACKUP_T2_MONTHLY,
+		];
```

Comment:
These are already defined in `@automattic/calypso-products` as `JETPACK_BACKUP_PRODUCTS`. May be simply import and use that? It will also remove one of the `if` blocks.

Reply:
I've looked into this, and found that the need to differentiate between daily and real-time backup features/products appears to make this necessary. The function should only `return true` if the user's site plan and cart product both include daily backup, or both include real-time backup, but not a mix of the two.

I've tried to see if there is a straightforward way of identifying if a backup product slug is a daily vs real-time product, but have not found any existing solutions that could be applied to `JETPACK_BACKUP_PRODUCTS` so far.

Reply:
Ah OK, thank you.